docker-compose.yml

```diff
@@ -1,5 +1,7 @@
 hawkular:
-  image: "hawkular/hawkular-services:latest"
+  # !! WARNING !! FIXME !! TODO !!
+  # Reset to hawkular/hawkular-services:latest before merging in master
+  image: "hawkular/hawkular-services:hawkular-1259"
```

Comment:
I think this can be updated to hawkular-services:latest before we forget about this

Reply:
pushed fix.